docker/runtime/csharpx/Kubeless.CSharpX/Extensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+
+namespace Kubeless.CSharpX
+{
+	public class Extensions
+	{
+		public HttpRequest Request { get; }

```

Comment:
Really great you implementation that adds `HttpRequest`, `HttpContext` and `HttpResponse` 👍